docs/source/quickstart.rst

```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
 Program your GoPiGo3
 ***********************
 
-For programming your `GoPiGo3`_ to do anything you want, read the instructions found here: `programming your robot`_.
+For programming your `GoPiGo3`_ to do anything you want, you can follow the instructions found here (`programming your robot`_) or *follow the rest of instructions found in this section*.
+
+To install or update the `GoPiGo3`_ library on your RaspberryPi, open a terminal or the command line and type the following command:
```

Comment:
isn't a terminal and the command line the exact same thing?

Reply:
Yeah it is.